doc/changelog.rst

```diff
@@ -1,5 +1,6 @@
 Changelog
 =========
+* :feature:`979` Change package of ``OrderedSet``
```

Comment:
Change Ordered set implementation from `orderedset` to `ordered-set`, as the former is not maintained anymore